plutus-tx-plugin/test/IntegerLiterals/NoStrict/NoNegativeLiterals/Spec.hs

```diff
@@ -0,0 +1,58 @@
+{-# LANGUAGE BangPatterns          #-}
+{-# LANGUAGE DataKinds             #-}
+{-# LANGUAGE FlexibleContexts      #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE NegativeLiterals      #-}
+{-# LANGUAGE OverloadedStrings     #-}
+{-# LANGUAGE ScopedTypeVariables   #-}
+{-# LANGUAGE TemplateHaskell       #-}
+{-# LANGUAGE TypeApplications      #-}
+
+{-# LANGUAGE NoNegativeLiterals    #-}
+{-# LANGUAGE NoStrict              #-}
+
+{-# OPTIONS_GHC -fplugin PlutusTx.Plugin #-}
+
+-- | This module tests that integer literals are handled correctly, when both @Strict@
+-- and @NegativeLiterals@ are off. These two extensions affect the Core we get.
+module IntegerLiterals.NoStrict.NoNegativeLiterals.Spec where
+
+import PlutusTx.Code
+import PlutusTx.Prelude qualified as PlutusTx
+import PlutusTx.Test
+import PlutusTx.TH (compile)
+
+import Test.Tasty.Extras
+
+tests :: TestNested
+tests = testNestedGhc "IntegerLiterals"
+  [ goldenPir "integerLiterals-NoStrict-NoNegativeLiterals" integerLiterals
```

Comment:
why not just include `NoStrict` as part of the test nesting?

Reply:
This way all golden files are in the same folder, which is a bit more convenient imo